apps/web/components/chat/message-item.tsx

```
@@ -731,20 +786,56 @@ function AttachmentGallery({ attachments }: { attachments: AttachmentRow[] }) {
         >
           {currentAttachment && (
             <div className="space-y-3">
-              <img src={currentAttachment.url} alt={currentAttachment.filename} className="w-full max-h-[75vh] object-contain rounded" />
+              <div
+                ref={imageContainerRef}
+                className="overflow-hidden rounded"
+                style={{ cursor: zoom > 1 ? "zoom-out" : "zoom-in" }}
+                onClick={handleImageClick}
+                onMouseMove={handleMouseMove}
+                onWheel={handleWheel}
+              >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)